README.md

```diff
@@ -7,3 +7,14 @@ npm i
 npm start
 curl -d 'https://www.nytimes.com/2018/06/11/technology/net-neutrality-repeal.html' -H 'Content-Type: text/plain' http://127.0.0.1:1969/search
 ```
+
+## With docker
+
+```
+git clone --recurse-submodules -j8 git@github.com:zotero/translation-server-v2.git
+cd translation-server-v2/
+docker build -t zotero/translation-server-v2:0.0.1 .
+docker run -it -d -p 1969:1969 zotero/translation-server-v2:0.0.1
```

Comment:
I don't think we need the version numbers for the dev steps.

Reply:
Right. I will replace it by latest.

For helping versioning and deployment, do you plan to create an automated build for this repository on the dockerhub zotero organization? (I can help if you want)
https://hub.docker.com/u/zotero/

(as it has been done for https://hub.docker.com/r/zotero/translation-server/ but I dont think it's an automated build ...)